inventory/hosts

```diff
@@ -63,6 +63,14 @@ openfoodnetwork.ca ansible_host=68.183.204.127
 [ca:children]
 ca-prod
 
+#------------------------------------------------------------------------------
+# Hungary
+[hu-prod]
```

Comment:
You can add this to the `europe` group to be managed by the global team.

Reply:
Added to Europe group, too.